src/components/pages/units-page/units-page.tsx

```diff
@@ -154,71 +184,106 @@ export const UnitsPage = () => {
+                </div>
               ))}
-              <Menu
-                id="unit-menu"
-                anchorEl={unitMenu.anchor}
-                keepMounted
-                open={Boolean(unitMenu.unit)}
-                onClose={handleUnitMenuClose}
+            </div>
+            {!defaultMusterConfiguration?.length && <span>No default requirements found.</span>}
+            <CardActions>
+              <Button
+                size="large"
+                onClick={showDefaultMusterConfig}
               >
-                <MenuItem onClick={editUnit}>Edit Unit</MenuItem>
-                <MenuItem onClick={deleteUnit}>Delete Unit</MenuItem>
-              </Menu>
-            </TableBody>
-          </Table>
-        </TableContainer>
+                Configure
+              </Button>
+              {defaultMusterSaved && (
+                <Typography variant="subtitle1" className={classes.defaultMusterSaved}>
+                  <CheckCircleIcon /> Changes saved successfully!
+                </Typography>
+              )}
+            </CardActions>
+          </CardContent>
+        </Card>
+        <Card>
+          <CardHeader title="Units" />
+          <CardActions>
+            <Button
+              size="large"
+              variant="text"
+              color="primary"
+              onClick={newUnit}
+              startIcon={<AddCircleIcon />}
+            >
+              Add New Unit
+            </Button>
+          </CardActions>
+          <CardContent>
+            <TableContainer className={classes.table}>
+              <Table aria-label="unit table">
+                <TableHead>
+                  <TableRow>
+                    <TableCell>ID</TableCell>
+                    <TableCell>Name</TableCell>
+                    <TableCell>Muster Requirements</TableCell>
+                    <TableCell />
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {units.map(unit => (
+                    <TableRow key={unit.id}>
+                      <TableCell>{unit.id}</TableCell>
+                      <TableCell>{unit.name}</TableCell>
+                      {unit.musterConfiguration.length > 0 && (
+                        <TableCell className={classes.musterConfiguration}>
+                          {unit.musterConfiguration.map((muster, index) => (
+                            <div key={JSON.stringify({ muster, index })}>
+                              {musterConfigurationToString(muster)}
+                            </div>
+                          ))}
+                        </TableCell>
+                      )}
+                      {unit.musterConfiguration.length === 0 && (
+                        <TableCell className={classes.noMusterConfiguration}>
+                          {defaultMusterConfiguration?.length ? (
+                            <span>* Using default muster requirements</span>
```

Comment:
There needs to be a bit more control over whether or not a unit uses the default muster configuration.  In the edit unit dialog there should be a checkbox for whether or not the unit uses the default (I believe this is in the mockups).  It should be possible for a unit to override the default to have no muster requirement.  My initial thoughts were if the unit had muster requirements set to `null` it would use the default, but if anything was defined then it would use that, so `[]` would mean to override the default with no muster requirements.

Reply:
MMmmmm... I missed that in the original mockup. Thanks, I'll go with the null vs [] approach.